File: tests/procperf/procperf.go
package main

import (
  "perfait"
  "fmt"
)

func main(){
  stopwatch := perfait.StopwatchNew()
  for i := 0; i < 100000000; i++ {}
  elapsedTime := stopwatch.ElapsedTime()
  fmt.Printf("%.6f\n", elapsedTime)
}

File: tests/procperf/procperf.cpp
#include <perfait.hpp>

int main(int argc, const char* argv[]){
  Perfait::Stopwatch stopwatch = Perfait::Stopwatch();
  for (int i = 0; i < 100000000; ++i){}
  double elapsedTime = stopwatch.ElapsedTime();
  printf("%.6f\n", elapsedTime);
  return EXIT_SUCCESS;
}
